cpp/test/IceGrid/session/Test.ice

```diff
@@ -1,16 +0,0 @@
-//
-// Copyright (c) ZeroC, Inc. All rights reserved.
-//
-
-#pragma once
-
-#include <Glacier2/PermissionsVerifier.ice>
-
-module Test
-{
-
-exception ExtendedPermissionDeniedException extends Glacier2::PermissionDeniedException
```

Comment:
The logic from removing this test is:
 - exceptions are now always encoded in Sliced format, so the old code should still work,
 - but there is no point in testing this Slice feature in the IceGrid/Session test

Reply:
This test no longer works since unkonwn slices for exceptions are no longer perserved with this PR. 

Glacier2 now forwards the sliced exception raised by the permission verifier.